DatadogSessionReplay/Tests/Utilities/SwiftExtensionsTests.swift

```diff
@@ -65,4 +72,11 @@ class TimeIntervalTests: XCTestCase {
         let uInt64MaxDate = Date(timeIntervalSinceReferenceDate: -.greatestFiniteMagnitude)
         XCTAssertEqual(uInt64MaxDate.timeIntervalSince1970.toInt64Milliseconds, Int64.min)
     }
+
+    func testTimeConvienience() {
+        let time = Date.mockDecember15th2019At10AMUTC().timeIntervalSince1970
+        XCTAssertEqual(time.minutes, time * 60)
+        XCTAssertEqual(time.hours, time * 60 * 60)
+        XCTAssertEqual(time.days, time * 60 * 60 * 24)
+    }
```

Comment:
Similar, we add these extensions as part of `Sources/Datadog/Utils/SwiftExtensions.swift`, but this test file relates to Session Replay's extensions. It needs to be moved.

Also, the underlying logic of this conversion distinguishes few edge cases that must be covered in tests.